package.json

```diff
@@ -30,9 +30,9 @@
     "check-supported-datatypes": "./scripts/check_supported_datatypes",
     "git-pre-commit": "./scripts/githooks/pre-commit",
     "lint": "eslint -c .eslintrc.cjs src scripts tests",
-    "lint-fix": "eslint -c .eslintrc.cjs src scripts tests --fix",
+    "lint-fix": "npm run lint -- --fix",
     "test": "./scripts/run_tests",
-    "prepublishOnly": "npm run lint && npm test && npm run update-dist",
+    "prepack": "npm run lint && npm test && npm run update-dist",
```

Comment:
Currently `npm pack` fails because of the update-dist script which also does git magic. Using it on a different branch than main doesn't work.

prepublishOnly would be executed before prepackaged which isn't that useful in this context. (See https://docs.npmjs.com/cli/v9/using-npm/scripts#life-cycle-operation-order)

Maybe the better way would be a combined build, lint & run tests command on `npm test`. Then prepack would just be `npm test` and testing always includes lints and typescript compilation.